### ISSUE TITLE
Crashes when importing flac files

### ISSUE THREAD

**Issue by pastarat** [LMMS/lmms#6504]
#### Steps to reproduce
Create new/open project, double click the sample track, import a flac file

#### Expected behavior
File is added to the track

#### Actual behavior
LMMS crashes almost immediately

#### Example of the issue
![lmmscrash](https://user-images.githubusercontent.com/112738262/188253414-596c8ae1-d99f-4172-84cb-e1674f6c8b8e.gif)

#### Details
- This happens exactly every time. I have never been able to import a flac file successfully
- This problem does not occur when importing wav or ogg files
- The file in the example is less than a single kilobyte. LMMS crashes no matter what size the file is
- It doesn't give me any error messages
- It doesn't show "Not Responding" 
- Other programs can process significantly larger flac files without lagging or crashing
**I have tried:**
- Using AudioFileProcessor instead of the sample track
- Updating LMMS
- Uninstalling and reinstalling LMMS
- Restarting my computer
- Importing smaller and/or shorter files
- Importing different files
- Using different projects
- Creating new projects
- Closing all other programs
- Deleting the lmmsrc.xml file

#### Hardware information
- Windows 10 Home 21H1
- 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz   2.42 GHz
- 8GB RAM (7.65 GB usable)
- 64-bit operating system, x64-based processor

#### Affected LMMS versions
1.2.0, 1.2.2 stable. Installed from https://lmms.io/download#windows

**Comment by Monospace-V** [comment]
Tried reproducing in 1.2.0. I can import a FLAC file just fine. I have checked, and it is _indeed_ a FLAC file I was using, not an mp3 or wav renamed .flac.
![image](https://user-images.githubusercontent.com/76674645/188277245-91036c57-ab9e-4e25-915f-0999baa83fb2.png)
This is quite an unusual issue.
![image](https://user-images.githubusercontent.com/76674645/188277419-f02fbae8-2b36-4da2-866c-b31da7ef0f82.png)

If it's not working with 1.2.0 or 1.2.2 it's unlikely to work with 1.3.

**Comment by pastarat** [comment]
I tried converting a video to wav but VLC corrupted it completely and it wouldn't play on any program. The flac files I tried to process had no bitrate when I checked their properties, but somehow would play. And then I was able to process a flac file from a different source successfully. I think it's VLC that's causing the problem, not LMMS. I did everything I could to find out what could be wrong...except importing a file that wasn't from VLC.
